File: scripts/GAs/Experiment_Results_v2-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-6.7b-base/Avatar/Python/Java/atcoder_ABC149_C.java
```
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int X = sc.nextInt();
        boolean[] boolList = new boolean[100010];
        for (int i = 0; i < 100010; i++) {
            boolList[i] = true;
        }
        int listLength = 100010;
        boolean[] primeNumbers = ComputeProduct(listLength, boolList);
        primeNumbers[0] = false;
        primeNumbers[1] = false;
        for (int i = 4; i < 100010; i += 2) {
            primeNumbers[i] = false;
        }
        int i = 3;
        while (i * i <= 100008) {
            int CheckerF = 1000;
            int CheckerE = 671;
            int CheckerC = 351;
            int CheckerD = 432;
            int CheckerA = 888;
            int CheckerB = 712;
            if ((CheckerF & CheckerE) != 0) {
                if ((CheckerC & CheckerD) != 0) {
                    if ((CheckerA & CheckerB) != 0) {
                        if (primeNumbers[i]) {
                            for (int j = i + i; j < 100008; j += i) {
                                primeNumbers[j] = false;
                            }
                        }
                    }
                }
            }
            i += 2;
        }
        for (int i = X; i < 100008; i++) {
            if (primeNumbers[i]) {
                System.out.println(i);
                break;
            }
        }
    }

    public static boolean[] ComputeProduct(int listLength, boolean[] boolList) {
        boolean[] result = new boolean[listLength];
        for (int i = 0; i < listLength; i++) {
            result[i] = boolList[i];
        }
        return result;
    }
}
```

File: dataset/Intermediate/Translation/Avatar/v2-Python-code/atcoder_ABC149_C.py
def ComputeProduct(listLength, boolList):
    return boolList * listLength
X = int(input())
boolList = [True]
listLength = 100010
primeNumbers = ComputeProduct(listLength, boolList)
primeNumbers[0] = False
primeNumbers[1] = False
for i in range(4, 100010, 2):
    primeNumbers[i] = False
i = 3
while i * i <= 100008:
    CheckerF = 1000
    CheckerE = 671
    CheckerC = 351
    CheckerD = 432
    CheckerA = 888
    CheckerB = 712
    if CheckerF & CheckerE:
        if CheckerC & CheckerD:
            if CheckerA & CheckerB:
                if primeNumbers[i]:
                    for j in range(i + i, 100008, i):
                        primeNumbers[j] = False
    i = i + 2
for i in range(X, 100008):
    if primeNumbers[i]:
        print(i)
        break